tests/sclT0009-unused-symbols/sym01.exec

```diff
@@ -0,0 +1,2 @@
+# format: exec
+exec:
```

Comment:
Oh wow! I was going to suggest adding the stdout test later. But you have done it already! :clap: 

Will do a final review tomorrow before merging.